lte/gateway/docker/services/python/Dockerfile

```diff
@@ -88,23 +88,24 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y \
   apt-transport-https \
   ca-certificates \
+  ethtool \
+  inetutils-ping \
+  iproute2 \
+  iptables \
+  isc-dhcp-client \
+  linux-headers-generic \
+  net-tools \
   netcat \
-  sudo \
-  python3-pip \
-  python3-venv \
-  virtualenv \
+  nghttp2-proxy \
   python3-eventlet \
-  python3-pystemd \
   python3-jinja2 \
-  nghttp2-proxy \
-  net-tools \
-  inetutils-ping \
+  python3-pip \
+  python3-pystemd \
+  python3-venv \
   redis-server \
-  wget \
-  ethtool \
-  linux-headers-generic \
-  iptables \
-  iproute2
+  sudo \
+  virtualenv \
+  wget
```

Comment:
When sorting the list of packages it is not easy as a reviewer to spot changes in this list. I think it would be good to add a comment to this review explaining what has actually changed. Personally I'm a fan of not even mixing functional changes with cosmetic changes for reasons of review readability, but I guess opinions diverge on this topic ;)

Reply:
Summary of this diff: The package `isc-dhcp-client` has been added.

Reply:
Good point, I reverted the sorting to make it easier to see what happens

Reply:
Separation of functional and cosmetic changes is a good idea. But I do not see any issue with both changes in one PR, if one separates those changes out in different commits. Accompanied by a respective comment in the PR description, this works well in my experience.

Reply:
Keep in mind that all commits of a PR will be squashed when merging, so git history will be more difficult to read even when splitting into different commits in the same PR.

Reply:
- Looking at [a merge commit](https://github.com/magma/magma/commit/15c16eb0bd358553af909a2201f0b0b977724261), I see a reference to [the respective PR](https://github.com/magma/magma/pull/13804), where I can easily access [the single commits](https://github.com/magma/magma/pull/13804/commits/da02598bda0387ef97df335f2f1224064b221013). So I do not see an issue here at all.
- The frequency and amount hardship caused by a chaos state in the current code is much larger, than that of a combined merge commit for functional and cosmetic changes in the past.